.env.mainnetdev

```diff
@@ -1,7 +1,7 @@
 DEFAULT_TESTNET=mainnet
 # Generated using co.clabs.valora.dev and the debug keystore
```

Comment:
Can you update the comment to use the correct identifier? (same as the other file)